packages/ui-text/src/Text/index.js

```diff
@@ -53,24 +45,19 @@ class Text extends Component {
     as: PropTypes.elementType,
     children: PropTypes.node,
     /**
-     * One of: primary, secondary, brand, success, warning, danger, alert, primary-inverse, secondary-inverse
+     * Color of the text
      */
-    color: deprecated.deprecatePropValues(
```

Comment:
Since it is a breaking change, please mark it as such in the commit's breaking change block, so it appears in the changelog.

Reply:
done